sdk/python/kfp/compiler/pipeline_spec_builder.py

```diff
@@ -2079,8 +2082,8 @@ def collect_pipeline_signatures(root_dict: dict,
                 string = '#    ' + signature['name'] + ': ' + signature[
                     'parameterType']
                 if 'defaultValue' in signature:
-                    string += ' [Default: ' + str(
-                        signature['defaultValue']) + ']'
+                    string += ' [Default: ' + escape_linebreak_characters(
```

Comment:
```suggestion
                    string += ' [Default: ' + repr(
```

Reply:
I think this might be a simpler and more complete fix?

Reply:
Actually I'm not sure which one is more faithful, extra quotes vs. extra backslash. 
Extra quotes would be more consistent as it's universally applied on all strings. Extra backslash is less intrusive and only visible on certain cases.